src/timepicker/timepicker.js

```diff
@@ -51,14 +51,14 @@ angular.module('ui.bootstrap.timepicker', [])
   var hourStep = timepickerConfig.hourStep;
   if ($attrs.hourStep) {
     $scope.$parent.$watch($parse($attrs.hourStep), function(value) {
-      hourStep = parseInt(value, 10);
```

Comment:
I think `+value` would be better if `parseInt` is not sufficient to ward off this issue.